v3/src/components/case-table/case-table.scss

```
@@ -12,6 +12,7 @@ $table-body-font-size: 8pt;
     width: 100%;
     height: 100%;
     display: flex;
+    overflow-x: scroll;
```

```suggestion
    overflow-x: auto;
```
We don't need the scroll bar unless there's something to scroll.